### PR TITLE
Fixing #19

### LINKED ISSUE #19: Feed for flet UI is not resuming on failure
During my testing for some reason my system went into sleep mode and when relogged into system.

Feed to Flet UI is broken and it is never resuming. I had restart the process altogether loosing my current positions.

### PR DESCRIPTION
**Pull request by AnveshMora**
Tried to do some refactoring of code for zerodha code and fixing broken issue.
Closes #19

### CODE REVIEW

File: pyalgomate/brokers/__init__.py [hunk 1]
```diff
@@ -318,3 +269,17 @@ def getBroker(feed, api, broker, mode, capital=200000):
             brokerInstance = LiveBroker(api)
 
     return brokerInstance
+
+
+def getDefaultUnderlyings() -> list[str]:
+    return  ['NSE:NIFTY BANK']
```
Comment (NagarajuGunda) [inline pyalgomate/brokers/__init__.py:275]: This file is common for all brokers, Can you move this method to zerodha file if its specific to zerodha?

File: pyalgomate/brokers/zerodha/__init__.py [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+                underlying, nextWeekExpiry, ltp, 10, strikeDifference)
+        if "Monthly" in registerOptions:
+            optionSymbols += getOptionSymbols(
+                underlying, monthlyExpiry, ltp, 10, strikeDifference)
+
+    optionSymbols = list(dict.fromkeys(optionSymbols))
+
+    tokenMappings = getZerodhaTokensList(api, underlyings + optionSymbols)
+    
+    return ZerodhaLiveFeed(api, tokenMappings), api
```
Comment (NagarajuGunda) [inline pyalgomate/brokers/zerodha/__init__.py]: Can you add new lines at the end of new files?

**Review by NagarajuGunda — CHANGES_REQUESTED** [review]

File: pyalgomate/brokers/__init__.py (hunk 1 above)
Comment (AnveshMora) [reply pyalgomate/brokers/__init__.py:275]: I wanted to generalize the code. I have handled the change for finvasia to replace from : to |, similar logic can be applied to Kotak if needed.

File: pyalgomate/brokers/zerodha/__init__.py (hunk 2 above)
Comment (AnveshMora) [reply pyalgomate/brokers/zerodha/__init__.py]: Sure

**Review by NagarajuGunda — APPROVED** [review]

**Pull request merged**